src/sonic-host-services/scripts/hostcfgd

```diff
@@ -381,6 +382,11 @@ class HostConfigDaemon:
 
 
     def update_all_feature_states(self):
+        syslog.syslog(syslog.LOG_INFO,
+                      "Waiting for systemctl to finish initialization")
+        self.wait_till_system_init_done()
+        syslog.syslog(syslog.LOG_INFO, "starting update of feature states")
```

Comment:
I feel like these lines of code might be better off in `start()`, placed right before the call to this function. I feel like this gets hidden if done inside this function.

Reply:
moved to `start()`

Reply:
move to 'start()'